README.md

```diff
@@ -22,3 +22,10 @@ GC.gc(true)
 ```
 
 Avoid assuming a key exists in a `WeakKeyIdDict`, especially when allocation occurs. Keys can be removed any time garbage collection occurs. Favor the methods `get!` and `get` over `getindex`.
+
+
+## Credits
+
+Much of the credit for the implementation of WeakKeyIdDict goes to Maura Werder (@mauro3) for the initial [work in the julia repository](https://github.com/JuliaLang/julia/pull/28182), and [DataStructures](https://github.com/JuliaCollections/DataStructures.jl/pull/402). Max Horn (@fingolfin) then updated and debugged a version merged to [AlbstractAlgebra](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1419).
```

Comment:
```suggestion
Much of the credit for the implementation of WeakKeyIdDict goes to Mauro Werder (@mauro3) for the initial [work in the julia repository](https://github.com/JuliaLang/julia/pull/28182), and [DataStructures](https://github.com/JuliaCollections/DataStructures.jl/pull/402). Max Horn (@fingolfin) then updated and debugged a version merged to [AlbstractAlgebra](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1419).
```
At least that's the way the name is written on their webpage https://maurow.bitbucket.io/